monitorable/config/models/config.go

```diff
@@ -12,23 +12,23 @@ type (
 		Version *ConfigVersion `json:"version"`
 		Columns *int           `json:"columns"`
 		Zoom    *float32       `json:"zoom,omitempty"`
-		Tiles   []Tile         `json:"tiles"`
+		Tiles   []ConfigTile   `json:"tiles"`
 	}
 
-	Tile struct {
-		Type   models.TileType        `json:"type"`
-		Params map[string]interface{} `json:"params,omitempty"`
+	ConfigTile struct {
+		Type models.TileType `json:"type"`
 
 		Label      string `json:"label,omitempty"`
 		RowSpan    *int   `json:"rowSpan,omitempty"`
 		ColumnSpan *int   `json:"columnSpan,omitempty"`
 
-		Tiles           []Tile `json:"tiles,omitempty"`
-		URL             string `json:"url,omitempty"`
-		InitialMaxDelay *int   `json:"initialMaxDelay,omitempty"`
+		Tiles           []ConfigTile `json:"tiles,omitempty"`
+		URL             string       `json:"url,omitempty"`
+		InitialMaxDelay *int         `json:"initialMaxDelay,omitempty"`
 
-		// Used by config.hydrate only (will be removed before returning config to UI)
-		ConfigVariant string `json:"configVariant,omitempty"`
+		// Used to validate config and for create api url (will be removed before returning config to UI)
```

Comment:
for => to

Reply:
// Used to validate config and to create API URLs
// Will be removed before being returned to the UI